regression/pages/lms/lms_home_page.py

```diff
@@ -0,0 +1,18 @@
+"""
+LMS Home page
+"""
+from bok_choy.page_object import PageObject
+from regression.pages.lms import BASE_URL
+
+
+class LmsHome(PageObject):
```

Comment:
@Muddasser 
We tend to keep it separate because we are using stage and we won't be able to use most of the methods. I think that is what we decided. Do you want me to change it?